spdlog/src/sink/rotating_file_sink.rs

```diff
@@ -276,6 +289,20 @@ impl RotationPolicy {
                 }
             }
             Self::Hourly => {}
+            Self::Duration {
+                hours,
+                minutes,
+                seconds,
+            } => {
+                if *hours == 0 && *minutes == 0 && *seconds == 0
+                    || *minutes > 59
+                    || *seconds > 59 {
+                    return Err(format!(
+                        "policy 'duration' expect `(hours, minutes, seconds)` to be ([0, u32::MAX], [0, 59], [0, 59]) and greater then (0, 0, 0) but got ({}, {}, {})",
+                        *hours, *minutes, *seconds
+                    ));
+                }
+            }
```

Comment:
If we use `Duration` as the parameter, it has already ensured the value correctness, thanks to which we don't need to check it again in our code.

But I want to return an error here if `duration < Duration::mins(1)`, as I commented https://github.com/SpriteOvO/spdlog-rs/discussions/73#discussioncomment-10654024 before, a rotating policy of less than 1 minute doesn't make much sense.